rakefile.rb

```diff
@@ -47,8 +47,7 @@
     - [ ] attach nupkg
     - [ ] publish the release
 - [ ] push NuGet package
-- [ ] copy release notes from GitHub to NuGet
-- [ ] de-list pre-release or superseded buggy NuGet packages if present (copy any release notes forward to the new version)

```

Comment:
Agreed, I think we should continue to do that, but this time that consolidation will be in GitHub and not NuGet. I'll amend.